### ISSUE TITLE
How do I use it for Google Drive?

### ISSUE THREAD

**Issue by stevefan1999-personal** [wunderio/csi-rclone#12]
It seems like I cannot go past the token generation

**Comment by Snake4life** [comment]
did you find a solution?

**Comment by Jancis** [comment]
so the `drive-token` would not work? does it work with rclone?
https://rclone.org/drive/#drive-token

**Comment by PatrickHuetter** [comment]
@Jancis No, it's mostly an access and renew token (within a jwt token). Without this, you can't use dropbox or similar services that use OAuth2 since the token changes from time to time (gets renewed by rclone) and needs to be persisted somewhere.

**Comment by Jancis** [comment]
Oh, sounds like chicken and egg problem where You need to mount an rwx storage on all rclone plugin pods and share the token, but You don't have any :) 

I had some success with [nfs-subdir-external-provisioner](https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner) and a [custom nfs server.](https://github.com/wunderio/charts/pull/333/files) If You mount a storage with rclone in it, You might have more or less the same system (but the storage is mounted in a single pod then). If You customize the nfs server, You could store and persist the token.

**Comment by dvcrn** [comment]
1. Do `rclone config` on your local machine
2. Connect to the remote backend
3. `rclone config view` 
4. Copy config into `PersistenVolume`

**Comment by PatrickHuetter** [comment]
This won't work stable since refresh tokens change and it might work the first time but won't after a while or after restarting the backend pod after a token isn't active anymore. The best case would be to you optionally a RWX volume OR a secret that can be written via k8s API by the pod. I think the best option would be to use an RWX volume since most storage providers already provide that.

**Comment by stevefan1999-personal** [comment]
> This won't work stable since refresh tokens change and it might work the first time but won't after a while or after restarting the backend pod after a token isn't active anymore. The best case would be to you optionally a RWX volume OR a secret that can be written via k8s API by the pod. I think the best option would be to use an RWX volume since most storage providers already provide that. 

Agreed. We need a feedback mechanism.
